SPEC.md

```diff
@@ -105,13 +105,17 @@ Gyp4apdU7AXEwysEQIb034aPrTlpmxh90SnTZFs2DHOvCjCPPAmoWfuQUwPhSPRb
+  "SignedEntryTimestamp": "MEQCIFRc9xXIoqCxUPXnoYlWyoWL4hX2aGnQOf0fVbxlUXdFAiAUkUBkEv+6Q4T0hdTgcnFgPdEHLwDigT7QD0cUvgRaOw==",
+  "Body": "eyJhcGlWZXJzaW9uIjoiMC4wLjEiLCJzcGVjIjp7ImRhdGEiOnsiaGFzaCI6eyJhbGdvcml0aG0iOiJzaGEyNTYiLCJ2YWx1ZSI6Ijk4OGNhN2U3MGViNmY2NWRkMzQ1ZWFjYjM4MGM1YmM0NDA2MmM3MjJhNTI1MzlmYWQ3Mzk0Mzg2ZTAyYTIwNWMifX0sInNpZ25hdHVyZSI6eyJjb250ZW50IjoiTUVRQ0lHRndhL2tTNlprUFlUN3RzSzl5UVlLeDhTU1paektNeU5vS2xOS3ZNWDhCQWlCZGVTdXp1ZWo5bWo1SWhYZjF2Rll5TlVNQkRxTUtzcXluaVN3WWtudC9DZz09IiwiZm9ybWF0IjoieDUwOSIsInB1YmxpY0tleSI6eyJjb250ZW50IjoiTFMwdExTMUNSVWRKVGlCRFJWSlVTVVpKUTBGVVJTMHRMUzB0Q2sxSlNVTjFWRU5EUVd0RFowRjNTVUpCWjBsVlFVcENXRVYyTWxaVE5uQkVSRGhMYTBaVFVrVTRURWN4VTIxTmQwTm5XVWxMYjFwSmVtb3dSVUYzVFhjS1MycEZWazFDVFVkQk1WVkZRMmhOVFdNeWJHNWpNMUoyWTIxVmRWcEhWakpOVWtWM1JIZFpSRlpSVVVSRmQyaDZZVmRrZW1SSE9YbGFWRUZsUm5jd2VRcE5WRUV3VFhwQmVVMVVWWGRPUkdSaFJuY3dlVTFVUVRCTmVrRjVUV3BGZDA1RVpHRk5SVWw0U0hwQlpFSm5UbFpDUVc5TlJtNUNlV0ZZYkdoa01rWnJDbUZJWkdoUlIyUjJZakprYzFwVE5XcGlNakI0U0hwQlpFSm5UbFpDUVUxTlJtNUNlV0ZZYkdoa01rWnJZVWhrYUZGSFpIWmlNbVJ6V2xNMWFtSXlNSGNLVjFSQlZFSm5ZM0ZvYTJwUFVGRkpRa0puWjNGb2EycFBVRkZOUWtKM1RrTkJRVk12ZG5CdGRVMUdZUzlGWWpsT1JWazFhMGczWkdKSFRFZGFlVkl2Y1FvNU1FSnNOM2hzUm5OaVNVWTVTWEZCUW1ReEwwRkpWV0pqT0RScVkzbHZOa1Z6U21aRmFYZFRabkIxY2s5Wk5IbHFWVEJSV2s4dloyODBTVUpMYWtORENrRlRXWGRFWjFsRVZsSXdVRUZSU0M5Q1FWRkVRV2RsUVUxQ1RVZEJNVlZrU2xGUlRVMUJiMGREUTNOSFFWRlZSa0ozVFVSTlFYZEhRVEZWWkVWM1JVSUtMM2RSUTAxQlFYZElVVmxFVmxJd1QwSkNXVVZHUzNkeGVtVnpNbGRQU1M4MFEycFVkMnRLWW5Wck0wRmthMDVzVFVJNFIwRXhWV1JKZDFGWlRVSmhRUXBHVFdwR1NGRkNRbTFwVVhCTmJFVnJObmN5ZFZOMU1VdENkRkJ6VFVsSFRrSm5aM0pDWjBWR1FsRmpRa0ZSVTBKblJFSXJUVWgzUjBORGMwZEJVVlZHQ2tKNlFVTm9ia0p2WkVoU2QwOXBPSFpqU0Vwd1pHMUdNRnBYVG1oTVYwNTJZbTVTYkdKdVVYUk9ha0Y2V20xVk0xcFVZM1JOUkVGM1RVTXdlVTFxU1RNS1RGZEtiVTU2VlhSYWFsSnRUbGRWTkUxSFVYbFBWRlV3VEc1T01HSXpTbWhhTWxWMVdqSTVkbG95ZUd4WldFSndZM2sxYW1JeU1IWlpNa1Y2VG0xRmVBcGFWR3N5VFdwUmVWbHFiRzFaTWtsNFRrUlpkbGt5UlhWWk0wb3dUVU5GUjBFeFZXUkZVVkZoVFVKcFFrWnVRbmxoV0d4b1pESkdhMkZJWkdoUlIyUjJDbUl5WkhOYVV6VnFZakl3ZDBObldVbExiMXBKZW1vd1JVRjNUVVJhZDBGM1drRkpkMGhqUjBWVFprMUtPRk41UmsxWk1GZHJhMHd4Wm1aNVdHcERlQ3NLVVZoUVRUZFFZbkEyZDJWWFpYUmtabU4zUzBoelRFNU1NMDFTTTJwbGFXeG9iV0p2UVdwQmVVcGtiamxQVm5sVllrRkJPVXR0VUdoa1kwcDVTRlIwV1Fwb1NFZzBObXBzUjJNMFJXMURSVXBwU2sxNmVuWm5WamRSYmxOeU5HaFJlRWRoYjFWQlowRTlDaTB0TFMwdFJVNUVJRU5GVWxSSlJrbERRVlJGTFMwdExTMEsifX19LCJraW5kIjoicmVrb3JkIn0=",
+  "IntegratedTime": 1619819449,
+  "LogIndex": 3460
 }
 ```
   The following are REQUIRED properties of the bundle:
     - The `SignedEntryTimestamp` is a rekor-signed signature over the logIndex, body and integratedTime of the Rekor Log Entry
-    - The `CanonicalizedPayload` is a JSON document following RFC 8785 rules consisting of the logIndex, body and integratedTime of the Rekor Log Entry
+    - The `Body` is the body of the Rekor Log Entry
+    - The `IntegratedTime` is the time the log entry was integrated into the transparency log
```

Comment:
Specify the format - I think it's just a UNIX timestamp right?

Reply:
👍🏽